Dockerfile.mustgather

```diff
@@ -1,10 +1,9 @@
-FROM registry.ci.openshift.org/ocp/4.8:must-gather AS builder
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.8 AS builder
```

Comment:
svc.ci.* does not work, please remove `svc.` (+ the same below and elsewhere in this project)